dist/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Wbpack Exercise</title><script defer="defer" src="index.bundle.js"></script></head><body><h1>Hello webpack!</h1></body></html>
```

Comment:
Kindly build your app for development or production. You can reference the webpack documentation or run (npm run build) in the terminal of your text editor.

Reply:
Thanks Seko 👋